data/libs/Equipment.lua

```diff
@@ -755,6 +755,24 @@ misc.hull_autorepair = EquipType.New({
 	l10n_key="HULL_AUTOREPAIR", slots="hull_autorepair", price=16000,
 	capabilities={mass=40, hull_autorepair=1}, purchasable=true, tech_level="MILITARY"
 })
+misc.thrusters_basic = EquipType.New({
+	l10n_key="THRUSTERS_BASIC", slots="thruster", price=30,
+	tech_level=1,
+	capabilities={mass=0, thruster_power=1}, purchasable=true,
+	icon_name="thrusters_basic"
```

Comment:
Feels like the `power_band` should be set in the `<ship>.json` files then? Or that is a future commit?